SimPEG/electromagnetics/static/resistivity/receivers.py

```diff
@@ -1,5 +1,6 @@
 import numpy as np
 from ....utils.code_utils import deprecate_class
+from discretize.utils import Identity
```

Comment:
I did not know this exists :)

Reply:
There is an Identity, but I think I actually don't need to import it here...